### ISSUE TITLE
It's unclear that premium extensions require a license

### ISSUE THREAD

**Issue by sybrew** [sybrew/the-seo-framework#729]
Currently, it's unclear why an extension cannot be activated. This must be addressed.

I recall I removed the "Premium/Free" tags when we discontinued the "Essentials" tier; I should have added something in its place.

A on-hover tooltip on the activation button would likely be most helpful.

![Image](https://github.com/user-attachments/assets/396a76ae-6196-478b-bed2-dbbfa28fb8be)

**Comment by sybrew** [comment]
Note that extensions cannot be activated when:
1. The license is invalid
2. No license is available
3. The extension isn't compatible with the current PHP version
4. The extension isn't compatible with the current WP version
5. A combination of the above
